apps/src/applab/designMode.js

```diff
@@ -508,7 +523,9 @@ function makeDraggable (jqueryElements) {
 
     wrapper.css({
       top: elm.css('top'),
-      left: elm.css('left')
+      left: elm.css('left'),
+      width: parseInt(elm[0].style.width, 10) + parseInt(elm[0].style.margin, 10),
+      height: parseInt(elm[0].style.height, 10) + parseInt(elm[0].style.margin, 10)
```

Comment:
Where does the margin come from?  It might be better to remove the margin instead of having to work around it here.